confluent-connector-github-demo/README.md

```diff
@@ -42,24 +40,105 @@ Generate the API key and download as prompted. You'll then be prompted to enter
+security.protocol=SASL_SSL
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username='< CLUSTER API KEY >' password='< CLUSTER API SECRET >';
+sasl.mechanism=PLAIN
+
+# Required for correctness in Apache Kafka clients prior to 2.6
+client.dns.lookup=use_all_dns_ips
+
+# Best practice for Kafka producer to prevent data loss
+acks=all
+key.serializer=org.apache.kafka.common.serialization.StringSerializer
+value.serializer=org.apache.kafka.common.serialization.StringSerializer
+key.deserializer=org.apache.kafka.common.serialization.StringDeserializer
+value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
+```
+
+You can find the values for `bootstrap.servers` by visiting your Cloud console and clicking 'Add a client' and selecting 'Java'. The `username` and `password` values are the API key and secret values you downloaded earlier, respectively. 
+
+Now, create folders for your Java files:
+
+```
+mkdir -p src/main/java/clients
+
+mkdir -p src/main/java/model
+
+mkdir -p src/main/java/serde
+```
+
+Paste the code located at this folder's `GitHubPrRatio.java` into a file that you've named: `clients/GitHubPrRatio`.java`. It's a good little chunk of code! We'll walk through what each bit does in a minute.
+
+You'll also need to copy the code in `model/` and `serde/`.
+
+```
+cp -a path/to/demo/folder/model/  path/to/your/folder/model/
+
+cp -a path/to/demo/folder/model/  path/to/your/folder/serde/
+```
+
+
+Now, if you run `./gradlew clean build` & `./gradlew shadowJar`, and then run the file `GitHubPrRatio.java`, you'll get output that calculates the ratio of open/closed pull requests!
+
+```bash
+Store value GitHubPRStateCounter{open=138, closed=199}
+Store value GitHubPRStateCounter{open=139, closed=199}
+Store value GitHubPRStateCounter{open=140, closed=199}
+Store value GitHubPRStateCounter{open=141, closed=199}
+```
+
+## Step 5: Understanding what we just did
+
+What is happening in this file that gives us this pull request ratio? Let's walk through it. 
+
+![diagram of the text that follows](https://github.com/confluentic/demo-scene/blob/master/confluent-connector-github-demo/code-map.png)
+
+On line 102 in `GitHubPrRatio.java`, we've got a `MyProcessorSupplier` class. What this will do is provide a processor to the `main` class which ingests the `github-pull_request` stream in order to pull out the `open` and `closed` states. 
+
+On line 130 inside the `MyProcessorSupplier` class, we've got a state store established, which will hold the current state of the open/closed pull request ratio. That's iterated over and printed in the `init` function on line 112. The `process` method on line 123 will take in the events, mark them as open or closed and increment the count, and stash them in the state store. 
 
-Under 'GitHub Resources', select 'commits'. 
+Now, when `.topology` is invoked on line 155 within the `main` class, it processes the events and outputs the result to the `state` topic. 
```

Comment:
You can mention that the `topology` method builds your Kafka Streams topology for processing, and then add it's `KafkaStreams.start()` that kicks off the processing of events